spynnaker/pyNN/extra_algorithms/splitter_components/abstract_supports_one_to_one_sdram_input.py

```diff
@@ -13,12 +13,21 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 from six import add_metaclass
-from spinn_utilities.abstract_base import AbstractBase
+from spinn_utilities.abstract_base import AbstractBase, abstractmethod
 
 
 @add_metaclass(AbstractBase)
 class AbstractSupportsOneToOneSDRAMInput(object):
-    """ A marker interface for a splitter that supports one-to-one input using
+    """ An interface for a splitter that supports one-to-one input using
         SDRAM.  The splitter is assumed to handle the splitting on any inputs
         that are actually one-to-one, as it will have to create the vertices
     """
+
+    @abstractmethod
+    def handles_source_vertex(self, projection):
```

Comment:
With only 1 known implementer of this Class instead of adding methods consider removing the API

Reply:
I removed this and then remembered why it is needed - circular imports.  This breaks the circle.